tests/fixtures/rules/mustache-spacing/invalid/always/snippet-render01-output.svelte

```diff
@@ -0,0 +1,4 @@
+<!-- prettier-ignore -->
+{ #snippet foo() }
```

Comment:
Also I don't think we need space before `{`.

```suggestion
{#snippet foo()}
```